lib/read-stream.js

```diff
@@ -143,15 +156,17 @@ class ReadStream extends stream.Readable {
       this._readBlock()
     }
 
-  }
+  },
 
   destroy() {
     fs.close( this.fd, ( error ) => {
       this.fd = null
       stream.Readable.prototype.destroy.call( this )
     })
-  }
+  },
 
 }
 
+inherit( ReadStream, stream.Readable )
```

Comment:
Nope, `util.inherits()` does weird stuff, and also works differently in different versions of node – which is why I wrote [bloodline](https://github.com/jhermsmeier/node-bloodline) back then. Bloodline is a close as I could get to native inheritance in JS, and generally everything should just work, from `instanceof` checks to extending with the new ES6 classes (even when inheriting a deeper prototype chain).